File: patch_auth.py
import re

files = ['clients.html', 'history.html']
for file in files:
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()
    
    # Check if it already has the auth section
    if 'id="sidebarAuthSection"' not in content:
        auth_html = """                </a>
                <div class="sidebar-divider"></div>
                <div id="sidebarAuthSection">
                    <button class="btn btn-primary sidebar-btn" onclick="window.app && window.app.showAuthModal && window.app.showAuthModal()" data-i18n="nav.signIn">Sign In / Sign Up</button>
                </div>"""
        
        # Replace the end of the pricing link
        content = re.sub(
            r'<span>Pricing</span>\s*</a>',
            r'<span>Pricing</span>\n' + auth_html,
            content
        )
        
        with open(file, 'w', encoding='utf-8') as f:
            f.write(content)
        print(f"Patched {file}")
    else:
        print(f"{file} already has auth section")

File: patch_auth.js
const fs = require('fs');

const files = ['clients.html', 'history.html'];
for (const file of files) {
    let content = fs.readFileSync(file, 'utf8');
    
    // Check if it already has the auth section
    if (!content.includes('id="sidebarAuthSection"')) {
        const authHtml = `                </a>
                <div class="sidebar-divider"></div>
                <div id="sidebarAuthSection">
                    <button class="btn btn-primary sidebar-btn" onclick="window.app && window.app.showAuthModal && window.app.showAuthModal()" data-i18n="nav.signIn">Sign In / Sign Up</button>
                </div>`;
        
        // Replace the end of the pricing link
        content = content.replace(
            /<span>Pricing<\/span>\s*<\/a>/,
            `<span>Pricing</span>\n${authHtml}`
        );
        
        fs.writeFileSync(file, content, 'utf8');
        console.log(`Patched ${file}`);
    } else {
        console.log(`${file} already has auth section`);
    }
}
